tensorflow/contrib/gdr/gdr_worker.cc

```diff
@@ -145,4 +149,41 @@ void GdrWorker::GrpcRecvTensorAsync(CallOptions* opts,
       });
 }
 
+void GdrWorker::RecvBufAsync(CallOptions* opts, const RecvBufRequest* request,
+                             RecvBufResponse* response, StatusCallback done) {
+  // This is an RDMA enabled implementation augmenting grpc.
+  Status s = recv_tensor_recent_request_ids_.TrackUnique(
```

Comment:
It is a shared member for both `RecvTensor` and `RecvBuf` and named `recent_request_ids_` in `GrpcWorkerService`. See 6042b5d267f42d004087b44c29525951700579f9. @fdxmw added `recv_tensor_recent_request_ids_` to `GdrWorker` as its only had this RPC back then.

Would it be okay to remove `GdrWorker` specific `recv_tensor_recent_request_ids_` and change the field in `GrpcWorker` to protected? It might have styling concern though.

Reply:
I will change to `recent_request_ids_` as that's what was done for both `RecvBuf` and `RecvTensor` in gRPC here 69ea4dee6f75276a76c72c2be4068dcc15041c02.